app/components/member_list/index.js

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React, {PropTypes, PureComponent} from 'react';
+import {
+    ListView,
+    StyleSheet,
+    Text,
+    View
+} from 'react-native';
+
+import Client from 'service/client';
+import {displayUsername} from 'service/utils/user_utils';
+
+import MemberListRow from './member_list_row';
+
+const style = StyleSheet.create({
+    listView: {
+        flex: 1
+    },
+    sectionContainer: {
+        backgroundColor: '#eaeaea',
+        paddingLeft: 10,
+        paddingVertical: 2
+    },
+    sectionText: {
+        fontWeight: '600'
+    },
+    separator: {
+        height: 1,
+        flex: 1,
+        backgroundColor: '#eaeaea'
+    }
+});
+
+export default class MemberList extends PureComponent {
+    static propTypes = {
+        members: PropTypes.array.isRequired,
+        onRowPress: PropTypes.func,
+        onListEndReached: PropTypes.func,
+        onListEndReachedThreshold: PropTypes.number,
+        sections: PropTypes.bool,
+        preferences: PropTypes.object
+    }
+
+    static defaultProps = {
+        onListEndReached: () => true,
+        onListEndThreshold: 100,
```

Comment:
Why is the default 100? it seems like a big number for a default value, maybe 10?